ui/raidboss/data/06-ew/raid/p8s.ts

```diff
@@ -412,6 +421,16 @@ const triggerSet: TriggerSet<Data> = {
         if (safe0 === undefined)
           return;
 
+        // Blazing Foothills will have 4 safe spots
+        // However, it will only be East or West
+        if (data.trailblazeCount === 1) {
+          if (safe2 === 'outsideEast')
+            data.trailblazeTorchSafeZone = 'east';
+          if (safe2 === 'outsideWest')
+            data.trailblazeTorchSafeZone = 'west';
```

Comment:
I agree that what you have is correct, but I think it's better to just check everything for safety.  The other option is to filter out corners from safeKeys and check if there's only one key and it's outside east/west.

```suggestion
          if (safeKeys.length !== 3)
            return;
          if (safe0 === 'cornerNE' && safe1 === 'cornerSE' && safe2 === 'outsideEast')
            data.trailblazeTorchSafeZone = 'east';
          if (safe0 === 'cornerNW' && safe1 === 'cornerSW' && safe2 === 'outsideWest')
            data.trailblazeTorchSafeZone = 'west';
```